webpack.config.js

```diff
@@ -1,7 +1,19 @@
 const path = require("path");
+const TerserPlugin = require("terser-webpack-plugin"); // included as a dependency of webpack
 
 module.exports = {
   entry: "./extension/src/content/index.js",
+  optimization: {
+    minimize: true,
+    minimizer: [
+      new TerserPlugin({
+        terserOptions: {
+          // https://github.com/webpack-contrib/terser-webpack-plugin/issues/107
+          output: { ascii_only: true }
```

Comment:
I tried this myself before but webpack kept generating an extra file for some reason

Reply:
Fixed!